frame/mosaic-vault/src/lib.rs

```diff
@@ -0,0 +1,904 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
```

Comment:
please add //! comment to document this pallet

Reply:
removed

Reply:
resolved